.travis/runpytest.sh

```diff
@@ -14,9 +14,9 @@ set -e
 
 ME=$(basename $0)
 SCRIPTDIR=$(readlink -f $(dirname $0))
-TOPDIR=$(readlink -f ${SCRIPTDIR}/..)
 
 . ${SCRIPTDIR}/utils.sh
+. ${SCRIPTDIR}/config.sh
```

Comment:
The config does not seem to be user here, is it?

Reply:
Similar to the above, this is primarily for the sake of consistency.

Reply:
similar to above with pylint -  the function `lsr_setup_module_utils` is needed in order to run pytest - so runpytest.sh must source both utils.sh in order to pick up the definition of the shell function lsr_setup_module_utils, and config.sh in order to run the shell function